test/k8sT/DatapathConfiguration.go

```diff
@@ -193,6 +188,9 @@ var _ = Describe("K8sDatapathConfig", func() {
 
 	Context("PerEndpointRoute", func() {
 		It("Check connectivity with IPv6 disabled", func() {
+			// Flannel always disables IPv6, this test is a no-op in that case.
```

Comment:
The `It()` was wrong in this case but I assume the same applies for `PerEndpointRoutes` as well.